src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -1753,6 +1753,69 @@ internal PointerTypeSymbol CreatePointerTypeSymbol(TypeSymbol elementType)
 
         #endregion
 
+        #region Operations
+
+        /// <summary>
+        /// Gets the low-level operation corresponding to the method's body.
+        /// </summary>
+        /// <param name="method">The method symbol.</param>
+        /// <param name="cancellationToken">An optional cancellation token.</param>
+        /// <returns>The low-level operation corresponding to the method's body.</returns>
+        protected override IOperation GetOperationCore(IMethodSymbol method, CancellationToken cancellationToken = default)
+        {
+            IOperation result = null;
+            var csmethod = method.EnsureCSharpSymbolOrNull<IMethodSymbol, MethodSymbol>(nameof(method));
+
+            if (csmethod != null)
```

Comment:
>csmethod [](start = 16, length = 8)

Please add cast to object because symbols have user-defined comparison operator and we don't want it to be called here.